src/target/riscv/riscv-013.c

```diff
@@ -140,6 +142,10 @@ typedef struct {
 	/* The program buffer stores executable code. 0 is an illegal instruction,
 	 * so we use 0 to mean the cached value is invalid. */
 	uint32_t progbuf_cache[16];
+
+	/* Some operations are illegal when an abstract command is running.
+	 * The field is used to track this. */
+	bool abtractcs_busy;
```

Comment:
I have tried to expand the comment, to make it clearer to the reader what the purpose of this flag is. Please let me know if you like it.

Also a typo in the identifier was fixed: `abtractcs_busy` --> `abstractcs_busy`

Or even better, the item can be renamed to signify that the abstract command **may** be busy, e.g. `abstract_cmd_maybe_busy` (or similar).

```suggestion
	/* Some operations are illegal when an abstract command is running.
	 * The field is used to track whether the last command timed out, and abstractcs.busy
	 * may have remained set. In that case we may need to re-check the busy state 
	 * at the beginning of the next command. */
	bool abstract_cmd_maybe_busy;
```

Reply:
Thanks for the suggestion. This is much better wording!
Addressed.
Although I have changed the comment a bit:
`at the beginning of the next command` -> `before executing these operations` (I think it's a bit more specific).